protocol_connect.go

```diff
@@ -124,6 +126,14 @@ func (h *connectHandler) NewConn(
 	if failed == nil {
 		failed = checkServerStreamsCanFlush(h.Spec, responseWriter)
 	}
+	if failed == nil && h.RequireConnectProtocolHeader && request.Header.Get(connectHeaderProtocolVersion) == "" {
```

Comment:
+1: The way I see it, if `RequireConnectProtocolHeader` is not set, it should allow it to be omitted, but regardless of whether or not the flag is set, if it is NOT omitted, it should only allow `1`, because that's the only value the specification specifies today.

Reply:
Makes sense to me!

Reply:
Makes sense to me!